### PR TITLE
Move to FMDB 2.7.9

### PR DESCRIPTION
**Pull request by wmathurin**
Submodule points ccgus/fmdb at 2.7.9  instead of the archived fork

### CODE REVIEW

**Comment by wmathurin** [comment]
We will host the podspec in our specs repo until it becomes available on cocoapods.org. See https://github.com/forcedotcom/SalesforceMobileSDK-iOS-Specs/pull/37

**Comment by wmathurin** [comment]
I'll send a PR for the templates to have a   
```ruby
source 'https://github.com/wmathurin/SalesforceMobileSDK-iOS-Specs'
```

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "external/fmdb"]
 	path = external/fmdb
-	url = https://github.com/forcedotcom/fmdb
+	url = https://github.com/ccgus/fmdb
```
Comment (brandonpage) [inline .gitmodules:3]: 🎉

**Review by bbirman — APPROVED** [review]

**Review by brandonpage — APPROVED** [review]

**Comment by wmathurin** [comment]
Actually doing [this](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/3683/commits/a59d684d0d10c7dd30a6cdfc65585ae2d9c17614) will avoid having to host FMDB.podspec and adding a source in all the templates.

**Pull request merged**